.env.example

```diff
@@ -16,3 +16,6 @@ DB_PORT=5432
 DB_USERNAME=tanulo
 DB_PASSWORD=tanulo
 DB_DATABASE=tanulo
+
+# Don't touch this
+DB_URL="postgresql://${DB_USERNAME}:${DB_PASSWORD}@${DB_HOST}:${DB_PORT}/${DB_DATABASE}?schema=public"
```

Comment:
CI crashes, because this is missing.
Can we introduce a new env var, or I should copy this line into the CI config?

Reply:
I think you'll have to add it to both .env and the CI configs, since we'll need it in both places (CI uses the vars defined in the yaml, while live env uses the .env file).